### DISCUSSION TITLE
Changelog entry for v0.6.0

### DISCUSSION THREAD

**Discussion by paulrudy** [ExistentialAudio/BlackHole#784] [General]
I noticed that BlackHole has been updated to v0.6.0, but the [changelog](https://github.com/ExistentialAudio/BlackHole/blob/master/CHANGELOG.md) hasn't been updated. Would you consider sharing a summary of the changes in the new version?

**Comment by devinroth** [comment]
Sorry about that. I updated the changelog.

**Reply by paulrudy** [reply]
> Sorry about that. I updated the changelog. 

Thanks!
